CmfRoutingBundle.php

```diff
@@ -96,10 +96,21 @@ private function buildOrmCompilerPass(ContainerBuilder $container)
                     realpath(__DIR__.'/Resources/config/doctrine-orm') => 'Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Orm',
                 ),
                 array('cmf_routing.dynamic.persistence.orm.manager_name'),
-                'cmf_routing.backend_type_orm',
+                'cmf_routing.backend_type_orm_default',
                 array('CmfRoutingBundle' => 'Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Orm')
             )
         );
+
+        $container->addCompilerPass(
+            $doctrineOrmCompiler::createXmlMappingDriver(
+                array(
+                    realpath(__DIR__.'/Resources/config/doctrine-model') => 'Symfony\Cmf\Bundle\RoutingBundle\Model',
```

Comment:
As in most of cases i will extend base model class (or even CMF Route class) but with custom mapping done in my bundle. If it would be missing then whole mapping for all properties defined in model (mapped superclass) would need to be redeclared in other mapping. Or this config directory would need to be added to `config.yml` mapping.